### DISCUSSION TITLE
[Support]: Issues with export on iPad

### DISCUSSION THREAD

**Discussion by simondsmason** [blakeblackshear/frigate#13457] [General]
I am using iPad running latest iOS 17.6.1.  I notice in custom export of video I can’t get the time picker to show.  If I switch to another tab and come back I do get it to come up but it ignores anything I select.  Can provide more details but wasn’t sure if iPad was supported?

**Comment by NickM-27** [comment]
yes iPads are supported

**Reply by simondsmason** [reply]
So in this case this might be a bug?  Also another strange thing I noticed.  I tried adding frigate to Home Screen as I have done in the past.  So it creates an icon that I can click on to load it directly without having to go through safari.  I just did this with the latest version - open in safari, add to Home Screen, then use the icon on the Home Screen.  All I get is a white screen every time I try the icon on the Home Screen.  I deleted and tried to recreate but get the same thing every time.  It loads fine in safari directly.  Tried this in chrome and got the same behavior?  Not sure what is causing this to happen?

**Reply by NickM-27** [reply]
Looks like it's a bug, but for Safari not Frigate

Not sure either, I've been running it this way in my iPhone for quite a while

**Reply by simondsmason** [reply]
It was my iPad. Reboot fixed the white screen. However issue with picking date and time persists. Works fine in iPhone so perhaps apple will fix the iPad version at some point.

**Comment by ItsMeRitch** [comment]
Also having the same issue on Safari. Will attach a screen recording of issue. https://github.com/user-attachments/assets/35071469-e66e-45b1-b1a2-71a4663ecf02

**Reply by ItsMeRitch** [reply]
Also not a safari issue, tested on Chrome also and issue prevails 

https://github.com/user-attachments/assets/0618c888-a5ca-495c-b9d1-d693e8a32a17

**Reply by ItsMeRitch** [reply]
Tested in macOS and issue could not be replicated. Issue seems to be localised to iOS

**Reply by NickM-27** [reply]
Chrome is just a wrapper on Safari (webkit) in iOS / iPad OS, so it most likely is a Safari issue still

**Reply by ItsMeRitch** [reply]
> Chrome is just a wrapper on Safari (webkit) in iOS / iPad OS, so it most likely is a Safari issue still

Figured it out, it's because the selector is using 24 hour time but the display is using AM/PM. 

All times under 12 midday work fine for the hour, just nothing after 12. For minutes and seconds, all is fine. 

Replicated on desktop too. Selecting a time after midday using the dropdown, not the keyboard, results in it being converted incorrectly, eg, selecting 19:00:00 results in 07:00:00. 

@NickM-27, you want me to open a bug issue for this or anything? :)

**Reply by NickM-27** [reply]
> Figured it out, it's because the selector is using 24 hour time but the display is using AM/PM.

there's nothing we can do about that, the browser controls the time picker and the time input

**Comment by ItsMeRitch** [comment]
The time picker isn’t the issue, it’s that Frigate is trying to interpret the input as 12-hour. Can frigate not convert the input from 24-hour to 12-hour? Or if not, just use 24 hours for the customer export selection and then convert it to 12 or whatever frigates backend uses?

**Reply by NickM-27** [reply]
No, frigate does not interpret anything, it passes the data directly from the time picker output to the browsers Date class. Frigate works entirely in timestamps, not formatted dates

**Reply by ItsMeRitch** [reply]
> No, frigate does not interpret anything, it passes the data directly from the time picker output to the browsers Date class. Frigate works entirely in timestamps, not formatted dates

Perhaps a check could be put in place in the backend? I can replicate the issue on desktop chrome so not a safari issue

**Reply by NickM-27** [reply]
sorry, a check for what?
